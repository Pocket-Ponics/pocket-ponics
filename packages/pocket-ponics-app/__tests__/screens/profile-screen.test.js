import React from 'react'
import { shallow } from 'enzyme'

import ProfileScreen from '../../screens/profile-screen'

describe('Profile Screen', () => {
	test('renders correctly', () => {
		global.greenhouses = {}
		const tree = shallow(
			<ProfileScreen  navigation={{ getParam: jest.fn() }}/>
		).html()
		expect(tree).toMatchSnapshot()
	})
})