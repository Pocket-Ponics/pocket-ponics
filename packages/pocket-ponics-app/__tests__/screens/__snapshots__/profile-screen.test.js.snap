// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile Screen renders correctly 1`] = `"<View style=\\"flex:1\\"><View style=\\"background-color:#472600;color:#FFFFFF;padding:20px;flex-direction:column;flex:1\\"><Text style=\\"color:#FFFFFF;font-size:18px;margin-bottom:20px\\">Username: </Text><Text style=\\"color:#FFFFFF;font-size:18px;margin-bottom:20px\\">Greenhouses: 0</Text><View style=\\"background-color:#456E31;border-radius:5px;padding:10px;margin:10px;align-items:center;opacity:1\\"><Text style=\\"color:#FFFFFF;font-size:22px\\">Change password</Text></View><View style=\\"background-color:#456E31;border-radius:5px;padding:10px;margin:10px;align-items:center;opacity:1\\"><Text style=\\"color:#FFFFFF;font-size:22px\\">Logout</Text></View></View></View>"`;
